com.unity.mobile.notifications/Runtime/Android/Plugins/com/unity/androidnotifications/UnityNotificationManagerNougat.java

```diff
@@ -2,64 +2,50 @@
 
 import android.app.Activity;
 import android.app.Notification;
-import android.app.NotificationChannel;
-import android.app.NotificationManager;
 import android.content.Context;
 import android.content.Intent;
-import android.content.SharedPreferences;
 import android.os.Build;
 import android.support.annotation.Keep;
-import android.util.Log;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import static android.app.Notification.PRIORITY_DEFAULT;
-import static android.app.Notification.VISIBILITY_PUBLIC;
-import static android.app.NotificationManager.IMPORTANCE_NONE;
 
 @Keep
 public class UnityNotificationManagerNougat extends UnityNotificationManager {
-
-    public UnityNotificationManagerNougat(Context context, Activity activity)
-    {
+    public UnityNotificationManagerNougat(Context context, Activity activity) {
         super(context, activity);
     }
 
-    public static void sendNotificationNougat(Intent intent, Context context) {
-
-        Notification.Builder notificationBuilder = UnityNotificationManager.buildNotification(intent, context);
-        int id = intent.getIntExtra("id", -1);
+    // Send notification for Android N and above.
+    protected static void sendNotificationNougat(Context context, Intent intent) {
+        Notification.Builder notificationBuilder = UnityNotificationManager.buildNotification(context, intent);
 
+        // TODO: setGroup/setGroupSummary/setSortKey are introduced in API Level 20, why we check N here?
+        // https://developer.android.com/reference/android/app/Notification.Builder
+        // If this is something we should move to UnityNotificationManager.buildNotification(), we probably can remove
+        // UnityNotificationManagerNougat as we don't need to do something special for Android N.
```

Comment:
@todi1856 @aurimasc This is something I want to discuss with you guys. 
`setGroup/setGroupSummary/setSortKey` are introduced in API Level 20, **_why we check N here?_**  Do you guys have any idea? Here is the document from google: https://developer.android.com/reference/android/app/Notification.Builder
If this is something we should move to UnityNotificationManager.buildNotification(), we probably can remove
UnityNotificationManagerNougat as we don't need to do something special for Android N. 
How do you think?

Reply:
Just finished at the full code, and I think you're right should probably be removed , especially due this code

            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
                UnityNotificationManagerNougat.sendNotificationNougat(context, intent);
            } else {
                UnityNotificationManager.sendNotification(context, intent);
            }

Reply:
yeah, we're checking `setGroup/setGroupSummary/setSortKey` with the wrong version code, so nothing special for Android N.
I'll remove `UnityNotificationManagerNougat` and move some check to `UnityNotificationManagerOreo`.

Reply:
Talked with Tomas offline, we agreed we should remove UnityNotificationManagerNougat, and move some code into UnityNotificationManager & UnityNotificationManagerOreo.
In order to make the changes more trackable, I planned to do it in a separate pr.